core/scripts/vrfv2/testnet/main.go

```diff
@@ -854,6 +855,112 @@ func main() {
 		resp, err := coordinator.GetSubscription(nil, *subID)
 		helpers.PanicErr(err)
 		fmt.Println("sub id", *subID, "balance:", resp.Balance)
+	case "coordinator-withdrawable-tokens":
+		withdrawableTokensCmd := flag.NewFlagSet("coordinator-withdrawable-tokens", flag.ExitOnError)
+		coordinator := withdrawableTokensCmd.String("coordinator-address", "", "coordinator address")
+		oracle := withdrawableTokensCmd.String("oracle-address", "", "oracle address")
+		helpers.ParseArgs(withdrawableTokensCmd, os.Args[2:], "coordinator-address", "oracle-address")
+
+		coordinatorAddress := common.HexToAddress(*coordinator)
+		oracleAddress := common.HexToAddress(*oracle)
+		abi, err := vrf_coordinator_v2.VRFCoordinatorV2MetaData.GetAbi()
+		helpers.PanicErr(err)
+
+		isWithdrawable := func(amount *big.Int) bool {
+			data, err := abi.Pack("oracleWithdraw", oracleAddress /* this can be any address */, amount)
+			helpers.PanicErr(err)
+
+			_, err = ec.CallContract(context.Background(), ethereum.CallMsg{
+				From: oracleAddress,
```

Comment:
This call won't actually work if the tx sender doesn't match the oracle address specified here right?

Reply:
The script actually doesn't need any private keys! You do need to set this oracle address param to the oracle address, but do not need the oracle private key to run it since it's just a simulation.